home-service/pom.xml

```diff
@@ -50,6 +50,11 @@
             <artifactId>validation-api</artifactId>
             <version>2.0.1.Final</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.uuid</groupId>
+            <artifactId>java-uuid-generator</artifactId>
+            <version>4.0.1</version>
```

Comment:
Please move version to properties section.

Reply:
Moved